openmc/mesh.py

```diff
@@ -210,3 +227,90 @@ def get_mesh_xml(self):
             subelement.text = ' '.join(map(str, self._width))
 
         return element
+
+    def build_cells(self, bc=['reflective'] * 6):
+        """Generates a list of cells which mimic the mesh geometry.
```

Comment:
This will produce a lattice, with an infinite cell for every [i,j,k].  The user then has to fill in the materials for each.  Perhaps a distribcell would be better but I dont fully understand how to input/use them.